socorro/processor/mozilla_transform_rules.py

```diff
@@ -851,7 +849,14 @@ def _predicate(self, raw_crash, raw_dumps, processed_crash, proc_meta):
             # We apply this Rule only if the release channel is beta, because
             # beta versions are the only ones sending an "incorrect" version
             # number in their data.
-            return processed_crash['release_channel'].lower() == 'beta'
+            # 2017-06-14: Ohai! This is not true anymore! With the removal of
+            # the aurora channel, there is now a new type of build called
+            # "DevEdition", that is released on both the beta and the aurora
```

Comment:
Is it really released on both channels? Isn't DevEdition still just using the `aurora` update channel under the hood?